hack/istio/install-istio-via-istioctl.sh

```diff
@@ -328,7 +328,10 @@ if [ "${DELETE_ISTIO}" == "true" ]; then
   done
 
   echo Deleting Core Istio
-  ${ISTIOCTL} manifest generate --set profile=${CONFIG_PROFILE} ${MANIFEST_CONFIG_SETTINGS_TO_APPLY} | ${CLIENT_EXE} delete -f -
+  # Although the 'uninstall' command has been available since Istio 1.7 and is listed in the official istio doc,
+  # it's still available in 'experimental' (x) scope, so it might change in the future.
+  # The optional --purge flag will remove all Istio resources, including cluster-scoped resources that may be shared with other Istio control planes.
```

Comment:
> including cluster-scoped resources that may be shared with other Istio control planes.

So this means this script will destroy other istio installs. If I have multiple istio installs and I want to remove one of them, I can't do it. I think we want to support that use-case. So I suggest adding another option here... "--purge-uninstall" - its default should be "true" but allow it to be "false" in case a user has multiple istio installs and only wants to uninstall one of them.

Reply:
Pushed requested changes.